src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -189,4 +208,41 @@ public String getApiVersion() {
     public String getApiBase() {
         return apiBase;
     }
+
+    /**
+     * Execute an HTTP request for a given method and URL with this EasyPostClient object.
+     *
+     * @param method   the HTTP method to use
+     * @param endpoint the API endpoint to use
+     * @param params   the parameters to use
+     * @param clazz    the class to use for deserializing the response
+     * @param <T>      the type of object to return
+     * @return the deserialized response
+     */
+    public <T> T request(final Requestor.RequestMethod method, final String endpoint, final Map<String, Object> params,
```

Comment:
Question: why are we moving the `request` functions from `Requestor.java` to `EasyPostClient.java`? Do you need to make this change to create the `MockClient` class?